tests/org.jboss.tools.openshift.test/src/org/jboss/tools/openshift/test/common/ui/utils/ConnectionWizardPageModelTest.java

```diff
@@ -43,21 +45,34 @@
 			"https://www.redhat.com", "http://openshift.com", "https://10.1.2.2:8443" };
 	private Connection editedConnection = mockOS3Connection("andre.dietisheim@chapeuvermelho.pt", allHosts[0]);
 	private List<Connection> os3Connections = Arrays.asList(
-			mockOS3Connection("adietish@roterhut.ch", allHosts[0]),
-			mockOS3Connection("adietish@chapeaurouge.ch", allHosts[1]),
-			mockOS3Connection("adietish@redhat.ch", allHosts[2])
+				mockOS3Connection("adietish@roterhut.ch", allHosts[0]),
+				mockOS3Connection("adietish@chapeaurouge.ch", allHosts[1]),
+				mockOS3Connection("adietish@redhat.ch", allHosts[2])
 			);
 	private List<IConnection> otherConnections = Arrays.asList(
-			mockOtherConnection("adietish@cappellorosso.ch", allHosts[0]),
-			mockOtherConnection("adietish@sombrerorojo.es", allHosts[1])
+				mockOtherConnection("adietish@cappellorosso.ch", allHosts[0]),
+				mockOtherConnection("adietish@sombrerorojo.es", allHosts[1])
 			);
 	
 	@SuppressWarnings("unchecked")
 	private List<IConnection> allConnections = 	union(os3Connections, otherConnections);
 
+	private TestableConnectionWizardPageModel model;
+
+	@Before
+	public void setUp() {
+		this.model = new TestableConnectionWizardPageModel(
+				editedConnection, 
+				allConnections,
+				editedConnection.getClass(),
+				true,
+				wizardModel);
+
+	}
+	
 	@SuppressWarnings("unchecked")
 	@Test
-	public void getAllConnectionsShouldReturnAllConnectionsAndNewConnectionMarkerIfConnectionTypeIsNullAndConnectionChangeIsAllowed() {
+	public void getAllConnections_should_return_all_connections_and_NewConnectionMarker_if_connection_type_is_null_and_connection_change_is_allowed() {
```

Comment:
I find the "_"-based naming easier to read even though worse in IDE support (you cant have your cursor jump from part to part via Ctrl+cursor lef/right - it only works within camel cased labels, it can't deal with underscore separated names)